app/src/main/java/com/android/voyageur/ui/profile/ProfileScreen.kt

```diff
@@ -166,17 +167,48 @@ fun ProfileContent(
         Spacer(modifier = Modifier.height(20.dp))
 
         // Trip Invite Menu
-        TripInviteMenu(tripInvites = tripInvites, tripsViewModel = tripsViewModel)
+        TripInviteMenu(
+            tripInvites = tripInvites,
+            tripsViewModel = tripsViewModel,
+            userViewModel = userViewModel)
       }
 }
 
 @Composable
-fun TripInviteItem(tripInvite: TripInvite, tripsViewModel: TripsViewModel) {
+fun TripInviteItem(
+    tripInvite: TripInvite,
+    tripsViewModel: TripsViewModel,
+    userViewModel: UserViewModel
+) {
+  val tripName = remember { mutableStateOf("Loading trip...") }
+  val senderName = remember { mutableStateOf("Loading user...") }
+
+  // Fetch the trip and user details asynchronously
+  LaunchedEffect(tripInvite) {
```

Comment:
Nice implementation to get the trip and user details!

Reply:
Thank you very much!